tests/yam/verify_consistency_self_update.pm

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 SUSE LLC
+#
+# Summary: check that self updates can be correctly installed on a
+#          GM installation.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
+
+    zypper_call 'ar -f http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-INSTALLER:/15-SP5:/x86_64/update/ self-update';
+    # enable all repos so that we can install yast later on
+    zypper_call 'mr -e --all';
+    zypper_call 'dup';
```

Comment:
wondering if we need a zypper ref, but maybe not, because we don't have update repos...but what I really missing here before adding the self update repo,  for a reviewer is to have a view of the repos, `zypper lr --uri` or something like that.

Reply:
no need for refresh (if needed dup will take care of it). Ok for lr, will add it